src/App.tsx

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const searchFilter = (movie: Movie) => {
+    const { title, description } = movie;
+    const searchLowerCase = search.toLowerCase().trim();
+
+    return title.toLowerCase().includes(searchLowerCase)
```

Comment:
DRY
make a separate function for this & reuse it
Also, split this with variables